Procfile

```diff
@@ -1,5 +1,5 @@
 thin:                      bundle exec rails s -p $PORT
-sidekiq:                   bundle exec sidekiq -q miq_bot
+sidekiq:                   bundle exec sidekiq -L log/sidekiq.log -q miq_bot
```

Comment:
It's useful to specify it when you're developing locally so you don't have to manually tee or redirect the output to a log file.